_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -47,6 +47,10 @@ early March.
 A registration portal will be created shortly. Unlike previous years, there will be a
 small registration fee.
 
+## Plenary speaker
+
+Yurii Nesterov will give a plenary during the workshop, more details on the topic and schedule will be communicated shortly.
```

Comment:
Link to his bio?

Reply:
It doesn't seem he owns a personal page, should we use https://uclouvain.be/en/directories/yurii.nesterov or https://en.wikipedia.org/wiki/Yurii_Nesterov ?

Reply:
Probs the wikipedia?